app/controllers/admin/base_controller.rb

```diff
@@ -52,8 +52,16 @@ def forbid_editing_of_locked_documents
     end
   end
 
-  def preview_design_system_user?
-    current_user.can_preview_design_system?
+  def preview_design_system?(next_release: false)
```

Comment:
Just a thought but I wonder if this should be renamed `show_design_system` or similar now that it will also be present for a launch?

Reply:
Design System changes which have 'been released' for general use, will drop use of the `preview_design_system?` method entirely.

For example, see the changes I made to `edition_tags_controller.rb` when launching it for all users in Release 1.2: https://github.com/alphagov/whitehall/pull/6893/files#diff-a3de518365bb0b4073122ea2e28fc75bf1d2df1bb0f870c561e049bc52b02b4a

The example I gave in this PR's description:

```diff
def preview_design_system?(next_release: false)
-  current_user.can_preview_design_system? || (next_release && current_user.can_preview_next_release?)
+  current_user.can_preview_design_system? || next_release
end
```

Would only ever be a temporary/hacky way of quickly releasing the feature to users. But if used, this would need to be followed up by a PR akin the the one I've just linked as part of a "post-release tidy up" process.

So I think it's still appropriate to call this `preview_design_system` because its primary purpose is for previewing unreleased Design System changes (with that one temporary/hacky exception above).